packages/commons/src/logging/loggerMiddleware.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import * as express from "express";
+import { LoggerMetadata, logger } from "./index.js";
+
+export function loggerMiddleware(serviceName: string) {
+  return (req: any, res: any, next: express.NextFunction): void => {
+    const loggerMetadata: LoggerMetadata = {
+      serviceName,
+      userId: req.ctx?.authData?.userId,
+      organizationId: req.ctx?.authData?.organizationId,
+      correlationId: req.ctx?.correlationId,
+    };
+
+    const loggerInstance = logger(loggerMetadata);
+
+    loggerInstance.info(`Request ${req.method} ${req.url}`);
+    res.on("finish", () => {
+      loggerInstance.info(`Response ${res.statusCode} ${res.statusMessage}`);
```

Comment:
Here we have to decide what we want to log.

Reply:
Shouldn't we log the same info that we were logging with the previous middleware?

Reply:
The old configuration was temporary so we could decide here the final version

Reply:
few considerations:
- I like the idea of having one log before and one after the call, it's easier to read. On the other side, this would reflect in a 50% log increase for the GETs, which are the most frequent called endpoints. Since Cloudwatch costs are not negligible, and considering that the double log does not actually add information, I would keep only the log at the request completion
- considering that the logger is already adding all context details, we can keep the previous log format 
  `Request {method} {urlWithParameters} - Response {code}`
  Notes:
  - we need to include query and path parameters but not elements in the body, which can be sensitive
  - the previous log includes also the response status message. It does not actually add any information and we could remove it, but I've not strong opinions on this


cc @beetlecrunch